src/dawn/engine_dawn.h

```diff
@@ -15,6 +15,8 @@
 #ifndef SRC_DAWN_ENGINE_DAWN_H_
 #define SRC_DAWN_ENGINE_DAWN_H_
 
+#include <shaderc/shaderc.hpp>
```

Comment:
This shouldn't be needed. Shaderc should only be known to ShaderCompiler.{h|cpp}